phoenix-ios/phoenix-ios/BusinessManager.swift

```diff
@@ -182,14 +182,16 @@ class BusinessManager {
 		log.trace("loadWallet()")
 		assertMainThread()
 		
+		if (business.walletManager.isLoaded()) {
+			return false
+		}
+
```

Comment:
@robbiehanson Probably something you want to take a look at, I think it's correct but it's worth checking.

Reply:
It's still correct. We have multiple guards setup, but this is a good place to have defensive programming.